Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -96,6 +104,16 @@ async function run() {
         kubectlCli = new kubernetescli(kubeconfigfilePath);
         kubectlCli.login();
     }
+
+    if (command === "save") {
+        let endpoint = tl.getInput("azureSubscriptionEndpointForACR");
+        let authScheme = tl.getEndpointAuthorizationScheme(endpoint, false).toLowerCase()
+        let isWifAuth = authScheme === "workloadidentityfederation";
+
+        if (isWifAuth) {
```

Comment:
Why use isWifAuth instead of direct check like this?
if (authScheme === "workloadidentityfederation")

Reply:
Done